Combat Rules/1_Combat_Rules.txt

```diff
@@ -0,0 +1,392 @@
+
+	If these are your final miss chances, you would have to drop the remainder 
+and your slightly-above-average perception wouldn't make enough of a difference 
+to affect your miss chances!
+
+	But let's say you also have a Dot Sight that you can put on your SMG. The 
+Dot Sight gives a bonus -0.5 range bracket miss chance (among other benefits). 
+This means you now have the following range brackets:
+
+	(4 - 0.5 - 0.5), (5 - 0.5 - 0.5), (6 - 0.5 - 0.5)  =  3, 4, 5
+
+	Perfect! Now your Perception and Dot Sight bonuses, together, reduce your 
+range bracket miss chances by 1 point. If you don't have any other bonuses (from 
+other attachments, feats, or abilities) then you would write down 3, 4, 5 as 
+your range bracket miss chances.
+
+	If your gun has automatic and/or burst fire, you'd add its associated miss 
+chances to your range bracket miss chances and separately write down those 
+numbers (after all bonuses to range bracket miss chance and bonuses to automatic 
+or burst fire). In other words, if you have a ranged weapon with semi-auto, 
+burst, and automatic fire modes, then you should write down 3 sets of 3 miss
+chances; one set for each of the 3 fire modes. For example, let's now assume 
+your gun has a burst fire miss chance of 1 and an automatic fire miss
+chance of 3, your ranged bracket miss chances should look something like this...
+
+0-10m, 11-20m, 21-40m	range brackets
+  3,     4,      5    	semi-auto miss chances
+  4,     5,      6	burst fire miss chances
+  6,     7,      8	automatic fire miss chances
+
+
+
+===== Thrown Weapons =====
+
+	These weapons work similarly to guns in that attackers add their combat PER 
+mod to their accuracy, however, like melee attacks, if you roll a critical hit or 
+3+ above your Miss Chance, your attack bypasses any armor the target is wearing. 
+Thrown weapons benefit from Stealth Attacks normally (see basic rules).
+	
+	Thrown weapons use the same size categories as Melee weapons for 
+determining proficiency, though a Two-handed throwing weapon would be 
+rare indeed.
+
+===== Grenades =====
+
+	All grenades require an explosives skill of at least 6 to use.
+You can 'cook' a grenade in your hand for 1 action to make it impossible for an
+enemy to dodge.
+
+	Gain +2 Accuracy if you are throwing under half of your grenade's max 
+throwing distance. If you fail your throw within 1 miss chance of the minimum, 
+roll d4 meters for how off-target the grenade throw is. If you fail your throw 
+beyond 1 miss chance, roll a d10 instead.
+
+	All grenades have a Reflex Modifier: -10
+
+
+Stealth Attacks:
+
+	If you are hidden (undetected by an opponent) when you initiate an attack 
+action, you incur bonuses depending on the circumstances of the attack.
+
+*Base Damage is typically the damage of the weapon without any other character 
+bonuses, but Base Damage can be increased by certain classes and abilities.
+
+**Increases to 200 max damage at level 10.
```

Comment:
We seem to be missing some lines between 386 and 388.

> 	Ranged weapon stealth-attacks (including explosive grenades but not any 
other thrown weapons) have -1 miss chance to hit if 'adequate' aiming time is 
given (DM's discretion: how rushed you are, or how skilled you are at stealth, 
etc). Ranged weapon stealth-attacks gain +20 bonus damage.
>
>	If you stealth-attack a target in melee, your target has a Guard of 0. As
normal (see section, "Melee Combat"), if you roll 3+ above the opponent's Guard, 
you ignore the opponent's armor and deal full damage. This signifies that the 
target gets hit in whatever you deem to be the most vulnerable; i.e., the head, 
neck, or the chest (over the heart, etc). 

Edit: Ah, I found them in the other files. I would argue they should be kept in the rules subsection that references starting a fight, since they're less likely to occur in the middle. Otherwise, the footnotes should be moved to refer to the relevant paragraphs or terminated.

Edit again: On balance, I like terminating the footnotes, they seem vestigial. In the end I suspect all the stealth stuff will move to a separate Stealth Combat subsection when the Tech Stealth project rolls around, so things are probably fine as they are.